src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2016,6 +2016,30 @@ public void testNewHandler() throws Exception {
         verifyWarns(checkConfig, getPath("InputIndentationNewHandler.java"), expected);
     }
 
+    @Test
+    public void testChainedMethodWithBracketOnNewLine() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+
+        checkConfig.addAttribute("arrayInitIndent", "2");
+        checkConfig.addAttribute("basicOffset", "2");
+        checkConfig.addAttribute("braceAdjustment", "0");
+        checkConfig.addAttribute("caseIndent", "2");
+        checkConfig.addAttribute("forceStrictCondition", "false");
+        checkConfig.addAttribute("lineWrappingIndentation", "4");
+        checkConfig.addAttribute("tabWidth", "2");
+        checkConfig.addAttribute("throwsIndent", "2");
+        final String[] expected = {
+            "44: " + getCheckMessage(MSG_CHILD_ERROR, "method call", 6, 8),
+            "45: " + getCheckMessage(MSG_CHILD_ERROR, "method call", 8, 10),
+            "47: " + getCheckMessage(MSG_ERROR, "method call rparen", 6, 8),
+            "61: " + getCheckMessage(MSG_ERROR, "foo", 5, 8),
+            "75: " + getCheckMessage(MSG_ERROR, "if rcurly", 4, 6),
+            "77: " + getCheckMessage(MSG_CHILD_ERROR, "method def", 2, 4),
+        };
+        final String fileName = "InputIndentationChainedMethodWithBracketOnNewLine.java";
+        verifyWarns(checkConfig, getPath(fileName), expected);
```

Comment:
please add to some Input following cases:
1) https://alinkov.github.io/full_report/reports/diff/Orekit/index.html#A4
2) https://alinkov.github.io/full_report/reports/diff/Hbase/index.html#A1 (chained method aligned by '.')
3) https://alinkov.github.io/full_report/reports/diff/Hbase/index.html#A42 - multiple chained methods and last call ends with ';'

we should keep in out test base such interesting code, to depend less on diff regression and let our Inputs to catch the most.

Reply:
done